Can't work on the project

Using Julia 1.5,

I can't instantiate the environment...
```
(@v1.5) pkg> activate .
 Activating environment at `~/Playground/www.juliacon.org/Project.toml`

(www.juliacon.org) pkg> instantiate
ERROR: AssertionError: sourcepath !== nothing
Stacktrace:
 [1] is_package_downloaded at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/Operations.jl:1670 [inlined]
 [2] #12 at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/Operations.jl:113 [inlined]
 [3] _all(::Pkg.Operations.var"#12#13"{Pkg.Types.Context}, ::Array{Pkg.Types.PackageSpec,1}, ::Colon) at ./reduce.jl:828
 [4] #all#638 at ./reducedim.jl:735 [inlined]
 [5] all at ./reducedim.jl:735 [inlined]
 [6] is_instantiated at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/Operations.jl:113 [inlined]
 [7] instantiate(::Pkg.Types.Context; manifest::Nothing, update_registry::Bool, verbose::Bool, platform::Pkg.BinaryPlatforms.MacOS, kwargs::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:828
 [8] instantiate(::Pkg.Types.Context) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:795
 [9] #instantiate#169 at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:791 [inlined]
 [10] instantiate() at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/API.jl:791
 [11] do_cmd!(::Pkg.REPLMode.Command, ::REPL.LineEditREPL) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/REPLMode/REPLMode.jl:401
 [12] do_cmd(::REPL.LineEditREPL, ::String; do_rethrow::Bool) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/REPLMode/REPLMode.jl:382
 [13] do_cmd at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/REPLMode/REPLMode.jl:377 [inlined]
 [14] (::Pkg.REPLMode.var"#24#27"{REPL.LineEditREPL,REPL.LineEdit.Prompt})(::REPL.LineEdit.MIState, ::Base.GenericIOBuffer{Array{UInt8,1}}, ::Bool) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/Pkg/src/REPLMode/REPLMode.jl:546
 [15] #invokelatest#1 at ./essentials.jl:710 [inlined]
 [16] invokelatest at ./essentials.jl:709 [inlined]
 [17] run_interface(::REPL.Terminals.TextTerminal, ::REPL.LineEdit.ModalInterface, ::REPL.LineEdit.MIState) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/LineEdit.jl:2355
 [18] run_frontend(::REPL.LineEditREPL, ::REPL.REPLBackendRef) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.5/REPL/src/REPL.jl:1144
 [19] (::REPL.var"#38#42"{REPL.LineEditREPL,REPL.REPLBackendRef})() at ./task.jl:356
```

If I delete the Manifest.toml, instantiate, then try running the site...

```
www.juliacon.org git:(src) ✗ julia --project=. -e 'using Franklin; serve()'                              
✘ Couldn't find highlight.js (``/Users/travigd/.julia/artifacts/1f34757b37b4d557365669ed69ebb5253d959e20/bin/node` -e "require('highlight.js')"` failed).
→ It is required for pre-rendering highlight.js but is not necessary to run Franklin (cf docs).
→ After installing any missing component, please re-build the package (cf docs).
 Activating environment at `~/Playground/www.juliacon.org/Project.toml`
→ Initial full pass...
┌ Franklin Warning: in <README.md>
│ Encountered an issue processing 'README.md' in w.juliacon.org/_site.
│ Verify, then re-start the Franklin server.
│ The error is displayed below:
│ ErrorException("type Nothing has no field captures")
└
ERROR: type Nothing has no field captures
Stacktrace:
 [1] process_file(::Symbol, ::Pair{String,String}, ::String, ::Vararg{Any,N} where N) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/file_utils.jl:88
 [2] fd_fullpass(::NamedTuple{(:other, :infra, :md, :html, :literate),NTuple{5,Dict{Pair{String,String},Float64}}}) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/franklin.jl:240
 [3] serve(; clear::Bool, verb::Bool, port::Int64, single::Bool, prerender::Bool, nomess::Bool, is_final_pass::Bool, no_fail_prerender::Bool, eval_all::Bool, silent::Bool, cleanup::Bool, on_write::Franklin.var"#231#234", log::Bool, host::String, show_warnings::Bool, launch::Bool) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/franklin.jl:118
 [4] serve() at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/franklin.jl:66
 [5] top-level scope at none:1
caused by [exception 1]
type Nothing has no field captures
Stacktrace:
 [1] getproperty(::Nothing, ::Symbol) at ./Base.jl:33
 [2] curyear() at /Users/travigd/Playground/www.juliacon.org/utils.jl:35
 [3] hfun_curyear() at /Users/travigd/Playground/www.juliacon.org/utils.jl:38
 [4] top-level scope at none:1
 [5] eval at ./boot.jl:331 [inlined]
 [6] convert_html_fblock(::Franklin.HFun) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/functions.jl:14
 [7] process_html_qblocks(::String, ::Array{Franklin.AbstractBlock,1}, ::Int64, ::Int64) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:121
 [8] process_html_qblocks(::String, ::Array{Franklin.AbstractBlock,1}) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:101
 [9] convert_html(::String) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:22
 [10] hfun_insert(::Array{String,1}) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/functions.jl:101
 [11] top-level scope at none:1
 [12] eval at ./boot.jl:331 [inlined]
 [13] eval at /Users/travigd/.julia/packages/Franklin/X69Nd/src/Franklin.jl:1 [inlined]
 [14] convert_html_fblock(::Franklin.HFun) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/functions.jl:18
 [15] process_html_qblocks(::String, ::Array{Franklin.AbstractBlock,1}, ::Int64, ::Int64) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:121
 [16] process_html_cond(::String, ::Array{Franklin.AbstractBlock,1}, ::Int64) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/blocks.jl:149
 [17] process_html_qblocks(::String, ::Array{Franklin.AbstractBlock,1}, ::Int64, ::Int64) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:111
 [18] process_html_qblocks(::String, ::Array{Franklin.AbstractBlock,1}) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:101
 [19] convert_html(::String) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/converter/html/html.jl:22
 [20] map at ./tuple.jl:160 [inlined]
 [21] write_page(::String, ::String; head::String, pgfoot::String, foot::String) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/write_page.jl:96
 [22] convert_and_write(::String, ::String, ::String, ::String, ::String, ::String) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/write_page.jl:198
 [23] process_file_err(::Symbol, ::Pair{String,String}, ::String, ::String, ::String, ::Float64) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/file_utils.jl:113
 [24] process_file(::Symbol, ::Pair{String,String}, ::String, ::Vararg{Any,N} where N) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/file_utils.jl:75
 [25] fd_fullpass(::NamedTuple{(:other, :infra, :md, :html, :literate),NTuple{5,Dict{Pair{String,String},Float64}}}) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/franklin.jl:240
 [26] serve(; clear::Bool, verb::Bool, port::Int64, single::Bool, prerender::Bool, nomess::Bool, is_final_pass::Bool, no_fail_prerender::Bool, eval_all::Bool, silent::Bool, cleanup::Bool, on_write::Franklin.var"#231#234", log::Bool, host::String, show_warnings::Bool, launch::Bool) at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/franklin.jl:118
 [27] serve() at /Users/travigd/.julia/packages/Franklin/X69Nd/src/manager/franklin.jl:66
 [28] top-level scope at none:1
```

--- a/utils.jl
+++ b/utils.jl
@@ -32,10 +32,7 @@ end
 
 # NAV_YEAR tools
 
-function curyear() 
-   x = match(r"^\/?(20\d{2})\/", locvar(:fd_rpath))
-   x != nothing ? parse(Int, x.captures[1]) : 2021
-end
+curyear() = parse(Int, match(r"^[\/|\\]?(20\d{2})[\/||\\]", locvar(:fd_rpath)::String).captures[1])
 
 function hfun_curyear()
     return "$(curyear())"

